docs/tables/oci_identity_tenancy.md

```diff
@@ -0,0 +1,31 @@
+# Table: oci_identity_tenancy
+
+The tenancy is an Oracle Cloud Account given to you when you register for Oracle Public Cloud (OCI).
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  id,
+  retention_period_days,
+  description
+from
+  oci_identity_tenancy;
+```
+
+### List tenancies where retention period is set to 365 days
```

Comment:
We should show something as below

List tenancies where retention period is not set to 365 days
```sqlselect  name,  id,  retention_period_days,  home_region_keyfrom  oci_identity_tenancywhere  retention_period_days <> 365;```